src/test/unzip.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { expect } from "chai";
```

Comment:
This thorough testing is great! However, these seem like integration tests to me. They are longer running than is ideal for our unit tests, and they also occasionally flake due to timeouts when run locally. 

I think these tests would be appropriate as integration tests. Could you separate them into a different npm command `test:unzip`, and ensure that they are not run as part of `npm run test` ?